language/go/fix_test.go

```diff
@@ -564,8 +564,9 @@ proto_library(
     visibility = ["//visibility:public"],
 )
 
-go_grpc_library(
+go_proto_library(
     name = "foo_go_proto",
+    compilers = ["@io_bazel_rules_go//proto:go_grpc"],
     importpath = "example.com/repo",
```

Comment:
It might be helpful to add a test here demonstrating how a `go_proto_library` with custom, non-default compilers would behave. I.e. 

- 1 case with `go_grpc` + `my_custom_compiler_foo`.
- 1 case with `my_custom_compiler_foo`.